addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -145,6 +145,31 @@ $bronze: #eea91e;
     .forum_answer img, .question-block img {
         max-width: 100%
     }
+
+    .o_wforum_index_entry_tags {
+        @for $size from 1 through length($o-colors) {
+            & .o_tag_color_#{$size - 1} {
```

Comment:
Why the `&`?

Reply:
Maybe I misunderstood the utility here, but shouldn't it be like : `.o_wforum_index_entry_tags .o_tag_color_#` ?

Reply:
Removing the & will have the same effect